Dockerfile.buildtests

```diff
@@ -1,13 +1,13 @@
 # Alpine
 FROM alpine:3.10 AS buildtest-alpine310-static
-RUN apk add --no-cache git build-base autoconf automake libtool linux-headers glib-dev glib-static libcap-static libcap-dev libseccomp-dev
+RUN apk add --no-cache git build-base autoconf automake libtool linux-headers glib-dev glib-static libcap-static libcap-dev libseccomp-dev libslirp-dev
 COPY . /src
 WORKDIR /src
 RUN ./autogen.sh && ./configure LDFLAGS="-static" && make && cp -f slirp4netns /
 
 # Ubuntu
 FROM ubuntu:18.04 AS buildtest-ubuntu1804-common
-RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev
+RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev libslirp-dev
```

Comment:
Probably `libslirp-dev` won't be available in Ubuntu 18.04, can we compile it from the source?

Reply:
it depends, do you expect ubuntu 18.04 to upgrade to upcoming slirp4netns v0.5? If it's only going in 20.04, then we are good.

Reply:
At least we should make sure it is manually installable from src on 18.04

Reply:
A similar question for other and old distributions that lack libslirp. Can we expect them to package libslirp when they update to v0.5? In which case we can remove it from vendor, but we will need to build libslirp in CI for a while. Otherwise, we need to keep the vendoring fallback.

Reply:
I assume QEMU will soon require libslirp pkg, and all distros want to provide QEMU, so anyway they will provide libslirp pkg?

Older versions of distros will continue to legacy version of QEMU & slirp4netns (v0.4) with vendored libslirp.

Reply:
QEMU has a compatbility story, they want the release to build on old-stable of various distros, until 2y after the new LTS release is out. So libslirp will be vendored in qemu for a while longer, likely ~2y. (unless those distro add/package libslirp, which is unlikely for some of them like RHEL)

The question is does slirp4netns needs such compatibility with old distros?

Reply:
> The question is does slirp4netns needs such compatibility with old distros?

I think no, old distros without lbslirp >= 4.1 should continue shipping slirp4netns < 0.5.

cc @rootless-containers/slirp4netns-package-maintainers RFC

Reply:
`ubuntu:20.04` (already available in Docker Hub) supports libslirp 4.1.0: https://packages.ubuntu.com/focal/libslirp-dev

Reply:
I tend to agree. Maybe the patch wouldn't downright remove the internal copy, but make its use optional?

If it helps, I could upload a libslirp copy to a PPA so that the Dockerfile could access it conveniently. Alternatively, I guess the dockerfile could download and install libslirp from the internet (but that feels rather awkward to me).

Reply:
> `ubuntu:20.04` (already available in Docker Hub) supports libslirp 4.1.0: https://packages.ubuntu.com/focal/libslirp-dev

I've tested the dockerfile locally, the tests pass on `ubuntu:20.04` with my changes mentioned in https://github.com/rootless-containers/slirp4netns/pull/162#discussion_r379921257 applied.